tests/benchmark/results.md

```diff
@@ -0,0 +1,36 @@
+# Benchmark Results
+
+## Dataset Used
+The following datasets are used for running performance benchmarking on Astro SDK `load_file` feature.
+
+All of them are based on real external datasets. For reproducibility of the benchmark tests, they were copied to:
+`gs://astro-sdk/benchmark/`
+
+Within `gs://astro-sdk/benchmark/`, there are two paths:
+* `original`: files with original size
+* `trimmed`: files trimmed to meet the desired sizes (for example, 10 GB)
+
+|            | size   | format  | rows      | columns | path to trimmed file(s)                     | description                 |
+|------------|--------|---------|-----------|---------|---------------------------------------------|-----------------------------|
+| ten_kb     | 10 KB  | parquet | 160       | 8       | covid_overview/covid_overview_10kb.parquet  | UK Covid overview sample    |
+| hundred_kb | 100 KB | csv     | 748       | 9       | tate_britain/artist_data_100kb.csv          | Tate Gallery artist sample  |
+| ten_mb     | 10 MB  | csv     | 600,000   | 3       | imdb/title_ratings_10mb.csv                 | IMDB title ratings sample   |
+| hundred_mb | 100 MB | csv     | 139,519   | 199     | github/github_timeline_100mb.csv            | Github timeline sample      |
+| one_gb     | 1 GB   | ndjson  | 940,000   | 17 (*)  | stackoverflow/stackoverflow_posts_1g.ndjson | Stack Overflow posts sample |
+| five_gb    | 5 GB   | ndjson  | 7,530,243 | 7 (*)   | pypi/*                                      | PyPI downloads sample       |
+| ten_gb     | 10 GB  | ndjson  | 1,263,685 | 9 (*)   | github/github-archive/*                     | Github timeline sample      |
+(*) Nested JSON, this number represents just the root-level properties
```

Comment:
Perhaps we could just add a link to the [dataset.md](./dataset.md). Ideally, this page will focus only on the results - and we'll update the README to both references the Dataset & Results.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/464/commits/636ae9bc8dc33af20e6440504c9d4644d6a2ded7)